plutus-context-builder/README.md

```diff
@@ -0,0 +1,46 @@
+# `plutus-context-builder`
+
+## What is this?
+
+A utility library for creating `ScriptContext` with `Spending` or `Minting` `ScriptPurpose`. This provides a builder abstraction for assembling ScriptContext, for use in `tasty-plutus`, or somewhere else.
+
+## What can this do?
+
+To create `ScriptContext`, you can use functions like `outToPubKey`, `inFromValidator`, `mintedValue` etc. These functions create named `ContextBuilder` parts, that describe various parts of the transaction from a script point of view.
+
+Once you have a `ContextBuilder`, you can
+* modify parts of it by accessing them by name 
+* build `ScriptContext` using `spendingScriptContext` and `mintingScriptContext` functions
+
+## What does 'script point of view' mean?
+
+`ScriptContext` is always passed to one specific _SCRIPT_ (_VALIDATOR_ or _MINTINGPOLICY_) for one specific _CHECK_.
```

Comment:
You can probably not capitalize `SCRIPT`, `VALIDATOR` and `MINTINGPOLICY`; since they are in italics, it already indicates that they are 'special' in their meaning, and all-caps feels a bit like shouting.